source/clear-linux/guides/maintenance/swupd-mixer-integration.rst

```diff
@@ -0,0 +1,117 @@
+.. _swupd_mixer_integration:
+
+Adding custom packages and bundles
```

Comment:
I would suggest "Adding Bundles to Clear Linux".  We don't normally talk about "packages" in Clear.